### PR TITLE
Add Sidebar buttons and routing

### PR DESCRIPTION
**Pull request by scandycuz**
## What 

- Sets up sidebar buttons and routing

### Also

- Updates README
- Fixes WizardForm bug

## Look

Screen Shot 2022-05-07 at 2 19 39 PM

### CODE REVIEW

**Comment by j-mateo** [comment]
Task linked: [CU-2wea2c3 Sidebar menu routing](https://app.clickup.com/t/2wea2c3)

File: src/components/home/SideBarButton.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { FunctionComponent } from "react";
```
Comment (escobarjonatan) [inline src/components/home/SideBarButton.tsx:1]: I am starting to see there might be different button variations..... i wonder if it's a good idea to talk to design about giving us a list of potential button looks and we can have a single button component that takes in for example a "palette" prop that changes the looks of it to try to standardize buttons for our app.  I bring this up bc i saw some differences in the signup flow between mobile and desktop. Thoughts?

File: src/docs/projectStructure.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+## Project Structure
```
Comment (escobarjonatan) [inline src/docs/projectStructure.md:1]: very nice! 🔥

File: src/pages/home/wallet/Wallet.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { FunctionComponent } from "react";
```
Comment (escobarjonatan) [inline src/pages/home/wallet/Wallet.tsx:1]: Thank you for putting the initial parts to these pages 🙌🏽

File: src/pages/signUp/SignUp.tsx [hunk 4]
```diff
@@ -29,33 +28,43 @@ const SignUp: FunctionComponent = () => {
   };
 
   /**
-    * Password regex, it must contain the following:
-    * 8 characters, 1 uppercase letter, 1 lowercase letter and 1 number.
-  */
+   * Password regex, it must contain the following:
+   * 8 characters, 1 uppercase letter, 1 lowercase letter and 1 number.
+   */
```
[the remark this answers is not in the document]
Comment (escobarjonatan) [reply src/pages/signUp/SignUp.tsx:33]: I think this is correct, i didnt run prettier on this file after i changed it lol

**Review by escobarjonatan — APPROVED** [review]

**Pull request merged**